README.md

```diff
@@ -4,10 +4,10 @@ The purpose of this repository is to formalize the governance process that the I
 
 ## Table of Contents
 
-* [Main Governance Document](governance.md)
-* [Current Steering Council and Institutional Partners](people.md)
-* [New Subproject Incubation Process](newsubprojects.md)
-* [Process for Authoring Jupyter Related Academic Papers](papers.md)
+* [Main Governance Document](governance)
+* [Current Steering Council and Institutional Partners](people)
+* [New Subproject Incubation Process](newsubprojects)
+* [Process for Authoring Jupyter Related Academic Papers](papers)
```

Comment:
These links (and the other changed links) no longer work on GitHub. Is there a way to have them work in GitHub too?

Reply:
I don't think so if we use the README.md file as the landing page for the documentation. This is because (I think) sphinx wants you to strip trailing file suffixes. Another option would be to maintain two files: `README.md` and `intro.md` ...README would be unchanged from it's previous state (maybe with a link to the documentation) and `intro.md` would be written for the documentation itself. WDYT?

Reply:
Ugh. So we can't easily support both the crowd reading the book and the crowd using the source. I guess it makes sense to have different intro files, though there are other links that were changed too that will break between the two versions.

I suppose we should cater towards the book by default, and assume if you are reading the source you can figure out where files live by putting that extra extension on...

Reply:
So I'll resolve this conversation for now.

Reply:
lemme look into it a bit and see if there's a way in Sphinx to make it support file extensions...I agree this is sub-optimal (though I also agree that in general probably the built website will be the more common way people interact with these docs)